src/lib/moneyHelpers.ts

```diff
@@ -39,49 +39,46 @@ export const isCurrencySupported = (currency: string): boolean => {
 }
 
 /**
- * Builds price display text (e.g. "$100").
+ * Builds price display text (e.g. "US$100").
  */
 export const priceDisplayText = (
-  priceCents: number | [number, number],
+  priceCents: number,
   currency: string,
   format: string
 ): string => {
-  if (typeof priceCents === "number") {
-    return currencyPrefix(currency) + priceAmount(priceCents, currency, format)
-  }
-  return priceRangeDisplayText(priceCents[0], priceCents[1], currency, format)
+  return currencyPrefix(currency) + priceAmount(priceCents, currency, format)
 }
 
 /**
- * Builds price range display text (e.g. "$100–$200" or "VUV Vt100–Vt200")..
+ * Builds price range display text (e.g. "US$100–US$200" or "VUV Vt100–Vt200")..
  */
 export const priceRangeDisplayText = (
-  lowerPriceCents: number,
-  higherPriceCents: number,
+  lowerPriceCents: number | null,
+  higherPriceCents: number | null,
```

Comment:
[`price_cents`](https://github.com/artsy/gravity/blob/8b1259a48a655aee0d9aa929577c5d08ac8e7e42/app/models/util/for_sale.rb#L208-L220) array from Gravity could return unbounded on either end, for example [`[nil, 100000]`](https://github.com/artsy/gravity/blob/eaf860818d504442f5d666f3cf43667cc5116571/spec/models/domain/artwork_spec.rb#L2037-L2043). This updates the logic of price range display to behave in a similar way as the [money_helper](https://github.com/artsy/money_helper/blob/8fb2110afa122c4ae32ba557000d2c0521fb2bf8/spec/money_helper_spec.rb#L199-L205) gem.